berkeley-mobile/Resources/Covid/CovidResourceViewController.swift

```diff
@@ -0,0 +1,250 @@
+//
+//  CovidResourceViewController.swift
+//  berkeley-mobile
+//
+//  Created by Oscar Bjorkman on 10/31/20.
+//  Copyright © 2020 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+fileprivate let kCardPadding: UIEdgeInsets = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+fileprivate let kViewMargin: CGFloat = 16
+
+class CovidResourceViewController: UIViewController {
+    private var overviewStack: UIStackView!
+    private var overviewCard: CardView!
+    
+    private var valueOne: UILabel!
+    private var valueTwo: UILabel!
+    private var lastUpdated: UILabel!
+    
+    private var screeningUrl = "https://calcentral.berkeley.edu/"   // fallback url
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        covidOverview()
+        screeningCard()
+        
+        DataManager.shared.fetch(source: CovidResourceDataSource.self) { resourceEntries in
+            guard let covidData = resourceEntries[0] as? CovidResource else { return }
+                        
+            self.screeningUrl = covidData.dailyScreeningLink
+            self.valueOne.text = covidData.positivityRate
+            self.valueTwo.text = covidData.totalCases
+            self.lastUpdated.text = "Last Updated: \(covidData.lastUpdated)"
+        }
+    }
+}
+
+extension CovidResourceViewController {
+    func covidOverview() {
+        let card = CardView()
+        card.layoutMargins = kCardPadding
+        
+        view.addSubview(card)
+        
+        card.translatesAutoresizingMaskIntoConstraints = false
+        card.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: kViewMargin).isActive = true
+        card.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
+        card.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor).isActive = true
+        
+        let headerLabel = UILabel()
+        headerLabel.font = Font.bold(24)
+        headerLabel.text = "Testing Dashboard"
+        
+        card.addSubview(headerLabel)
+                
+        headerLabel.translatesAutoresizingMaskIntoConstraints = false
+        headerLabel.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
+        headerLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
+        headerLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
+        
+        let stack = UIStackView()
+        stack.axis = .horizontal
+        stack.distribution = .fillEqually
+        stack.alignment = .center
+        stack.spacing = 9
+        
+        let (uhsTests, uhsLabel) = createOverviewCard(cardHeader: "Positivity Rate", cardValue: 0)
+        let (positiveTests, positiveLabel) = createOverviewCard(cardHeader: "Positive Tests", cardValue: 0)
```

Comment:
When the data is still loading, these numbers will read 0. Could `cardValue` be an optional int or something so that it will display "N/A" instead while loading?